playbooks/ansible.cfg

```diff
@@ -11,5 +11,17 @@ forks = 15
 # SSH timeout
 timeout = 120
 
+# Set the path to the folder in os-ansible-deployment which holds the dynamic
+# inventory script
+inventory = /opt/os-ansible-deployment/playbooks/inventory/
```

Comment:
@hughsaunders I think that the user guides make use of /opt/os-ansible-deployment, which is why this convention was used throughout rpc-extras (you'll see it in the group vars too).